rpm/kernel-adaptation-pc.spec

```diff
@@ -3,8 +3,9 @@ Name:       kernel-adaptation-pc
 %define kernel_version_build %{version}-%{release}
 %define kernel_devel_dir %{_prefix}/src/kernels/%{kernel_version_build}
 %define kernel_version %{version}
+%define kernel_arch x86
 
-Summary:    Kernel Adaptation %{kernel_target_hw}
+Summary:    Kernel Adaptation PC
 Version:    3.6.11
 Release:    8
```

Comment:
While here we could reset these to 0

Reply:
No, this package requires it as it is. The above %define kernel_devel_dir relies on Version and Release having this exact format and Version matching the version defined in Makefile.